packages/web3/src/IStaticContractsInfo.ts

```diff
@@ -9,6 +9,7 @@ export interface BaseChainConfig {
         spaceOwner: Address
         baseRegistry: Address
         riverAirdrop?: Address
+        riverPointsFacet?: Address
```

Comment:
we don't put facet addresses here. @giuseppecrj or @shuhuiluo  is river points on a new diamond? What goes here?

Reply:
thanks for checking; I'm unfamiliar with this area.

Reply:
river points is on riverAirdrop

Reply:
the riverAirdrop addresses were never put in the right folder. I'm going to re-create my pr that copies everything over and get this fixed up today.

Reply:
Thank you – as far as I understand @texuf is taking a look. I doesn't look like riverAirdrop is actually hooked up to the dApp yet.